### DISCUSSION TITLE
Who's Hiring (April 2023)

### DISCUSSION THREAD

**Discussion by rdylina** [vercel/next.js#47832] [Show and tell]
I'm opening the discussion since it hasn't been opened it.

**Comment by rdylina** [comment]
REI Underwrite | Senior Full-Stack Engineer | Remote (US, Canada) | Contract | $10-14k per month DOE  

This is a startup in the real estate space. NDA required. I've done a fair amount of the base of the project, but I need help pulling it across the finish line to MVP. This is a month-to-month contract as an independent contractor, but it is possible it will lead to a full-time position if the product gains traction. There is plenty of work to do in the roadmap.

Stack/Packages involved:
NextJS 13 Stable
Stripe
ClerkDev Auth
Supabase
Tanstack Query
Tanstack Tables
GoogleMaps/Mapbox

TailwindCSS+HeadlessUI

**Reply by codepool867** [reply]
Dear Robert,

I am writing to express my interest in the Senior Full-Stack Engineer position at REI Underwrite. With my extensive experience in web development and expertise in NextJS, Stripe, ClerkDev Auth, Supabase, Tanstack Query, Tanstack Tables, GoogleMaps/Mapbox, and TailwindCSS+HeadlessUI, I am confident that I would make a valuable contribution to your team.

As a Senior Full-Stack Engineer, I understand the importance of pulling a project across the finish line to MVP. I have been involved in various stages of software development projects and have played a critical role in the completion of these projects. My ability to quickly understand and adapt to different technologies, along with my strong problem-solving skills, has allowed me to excel in a fast-paced startup environment.

I have experience working remotely as an independent contractor, and I am comfortable working on a month-to-month contract. I understand the importance of maintaining confidentiality and adhering to NDA requirements. I am committed to delivering high-quality work within the given timeline and budget, and I am always open to feedback and suggestions.

I am excited about the opportunity to work with your team and contribute to the success of your startup. Please find attached my resume and relevant work samples. Thank you for considering my application. I look forward to hearing from you soon.

https://strategy2earn.online/
[Salman Alampour.docx](https://github.com/vercel/next.js/files/11174004/Salman.Alampour.docx)
email: alampourztech@gmail.com

Best regards,
Salman

**Reply by thesigmadeveloper** [reply]
Dear Robert,

I am writing to express my interest in the REI Underwrite | Senior Full-Stack Engineer position that you have advertised. As an experienced full-stack developer with a proven track record of delivering successful projects, I am confident that I can make a significant contribution to your team.

I am excited about the opportunity to work on a project that is at the forefront of the real estate industry. Having worked on a range of projects in my career, I am confident that my skills in NextJS 13 Stable, Stripe, ClerkDev Auth, Supabase, Tanstack Query, Tanstack Tables, GoogleMaps/Mapbox, TailwindCSS, and HeadlessUI are a perfect match for this project.

I understand that you need help pulling the project across the finish line to MVP. With my skills in full-stack development, I am confident that I can help take the project to the next level. I have experience working with startups and I understand the need for confidentiality. I am happy to sign an NDA to ensure that all proprietary information is protected.

I am available to work remotely and am based in the US. I understand that this is a month-to-month contract as an independent contractor, but I am excited about the possibility of a full-time position if the product gains traction. I am confident that my experience in delivering successful projects can help ensure the project's success.

Thank you for considering my application. I look forward to discussing the opportunity with you further.

My email is: thesigmadev@gmail.com 

Gupta Aporve
